src/theme/ContactForm/enable-javascript.js

```diff
@@ -1,28 +1,20 @@
-/**
- * Copyright (c) The Libra Core Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
+import React from 'react';
 
-const React = require('react');
-
-
-const EnableJavaScript = ({ baseUrl }) => {
-  return (
-    <div id="disable-ad-block" className="modal">
-      <div className="disable-adblock">
-        <div className="inner">
-          <img src={`${baseUrl}img/ab-icon@2x.svg`} alt="Adblock icon" />
-          <h2>Please enable JavaScript!</h2>
-          <p>We use JavaScript to validate that you submitted a correct email address and URL.</p>
-          <div className="buttonWrapper">
-            <a id="disable-adblock-refresh" className="button secondary" href="#">
-              Refresh
-            </a>
-          </div>
+const EnableJavaScript = ({ baseUrl }) => (
+  <div id="disable-ad-block" className="modal">
+    <div className="disable-adblock">
+      <div className="inner">
+        <img src={`${baseUrl}img/ab-icon@2x.svg`} alt="Adblock icon" />
+        <h2>Please enable JavaScript!</h2>
```

Comment:
This will need to be changed before release. It's not JS that needs to be enabled, it's that ad blockers need to be disabled.

Reply:
That makes sense to me https://dfuzrdev.atlassian.net/browse/LIB-720